designer/client/pages/landingpage/Choice.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactElement, useState } from "react";
```

Comment:
Please rename the `landingpage` folder to `LandingPage`. Convention is that PascalCase folders indicate the presence of components. The case applied to the files is correct. 